client/components/banner/test/test.jsx

```diff
@@ -1,4 +1,7 @@
-/** @format */
```

Comment:
The convention is for the test file name to match the target file name, in this case `index.js` (you can drop the `x` in the extension since it's not strictly jsx :) )

Reply:
> not strictly jsx

Really? I thought the `<Banner { ...props } price={ 100 } />` meant jsx - what's the technical criteria?

Reply:
I should have qualified: _our_ (apparent) [convention](https://github.com/Automattic/wp-calypso/blob/master/docs/testing/unit-tests.md#folder-structure)

I believe the debate rages on, but I like to think that `.jsx` describes what the file is to the transpiler/app at a higher level. 

Saying that, no need to change it in this PR. It was just me typing aloud 😄